apps/condo/pages/property/index.tsx

```diff
@@ -32,12 +32,15 @@ import { useTableColumns } from '@condo/domains/property/hooks/useTableColumns'
 import { Property } from '@condo/domains/property/utils/clientSchema'
 
 import debounce from 'lodash/debounce'
+import { isEmpty } from 'lodash'
```

Comment:
```suggestion
import isEmpty from 'lodash/isEmpty'
```

i'm not sure but there is some problems with lodash full import when treeshaking is applyin by webpack in prod

Reply:
Done